src/Replies.cpp

```diff
@@ -803,3 +803,9 @@ string Replies::ERR_REGFAILED() {
     reply = "ERROR :Registration failed";
     return Message::create(reply);
 }
+
+std::string Replies::ERR_QUIT(Client* client) {
+    std::string reply;
+    reply = client->getName() + " quit";
```

Comment:
invalid format, the format is : 
```"ERROR :Goodbye !"```
since the message sent to the client is an ERROR message.
the client as parameter is not used.